packages/build/src/log/messages/plugins.js

```diff
@@ -38,6 +38,16 @@ const logFailPluginWarning = function (methodName, event) {
   )
 }
 
+const logPluginNodeVersionWarning = function ({ logs, pluginNames, userNodeVersion, currentNodeVersion }) {
+  logWarning(
+    logs,
+    `Warning: we have detected the following plugins ${pluginNames.join(
+      ', ',
+    )} have been installed locally or via package.json. The Node.js version used for the execution of this types of plugins in our build system will be changing soon (in your case ${userNodeVersion}) to always rely on the version we set for our system (version ${currentNodeVersion}). If you're relying on a plugin from our plugin list it almost certainly supports this, however if that is not the case please double check the plugin compatibility.
```

Comment:
I am wondering whether there would be a shorter way to express this, and also to emphasize the action call, something along the lines of:

```
Warning: Please ensure the following plugins are compatible with Node.js 12.18.0: ${pluginNames.join(',')}
Those plugins are currently run with Node.js 8.0.0. However, they will soon be run with Node.js 12.18.0 instead due to some upcoming changes in our build system. 
For more info ...
```

What do you think?

Reply:
Updated the text, let me know what you think 😄

Reply:
Looks good!